memegen/templates/image.html

```diff
@@ -1,27 +1,33 @@
 <!DOCTYPE html>
 <html>
-<head>
+  <head>
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@jasich">
+    <meta name="twitter:title" content="memegen.link">
+    <meta name="twitter:description" content="{{ title }}">
+    <meta name="twitter:image" content="{{ src | safe }}">
+    <meta name="twitter:image:alt" content="oh no!">
```

Comment:
Yeah I might remove it, I don't think it worked when the image failed to load.  Might only happen on specific error codes.